### PR TITLE
Three bug fixes for deployment

### PR DESCRIPTION
**Pull request by BradHN1**

### CODE REVIEW

**Comment by Davidjwkinghorn** [comment]
Does this fix the issue where changing the full name overwrites the profile icon chosen by a user?

**Comment by BradHN1** [comment]
Yes.

File: src/components/Pages/ProfilePage/EditingProfileForm.js [hunk 1]
```diff
@@ -46,11 +47,13 @@ class EditingProfileForm extends React.Component {
...
         full_name: this.state.full_name,
-        profile_picture: this.state.image,
         preferred_name: this.state.preferred_name,
       };
+      if (this.state.image) {
+        body.profile_picture = this.state.image;
+      }            
       this.setState({ loading: true });
-
+      console.log("onSubmit body", body)
```
Comment (frimpongopoku) [inline src/components/Pages/ProfilePage/EditingProfileForm.js:56]: 😄

File: src/components/Pages/ProfilePage/EditingProfileForm.js [hunk 2]
```diff
@@ -32,6 +32,7 @@ class EditingProfileForm extends React.Component {
 
   //updates the state when form elements are changed (not including image)
   onChange(event) {
+    console.log("onChange event.target", event.target)
```
Comment (frimpongopoku) [inline src/components/Pages/ProfilePage/EditingProfileForm.js:35]: 😄

**Review by frimpongopoku — APPROVED** [review]
Just console logs

**Pull request merged**